src/expr/scalar/func.rs

```diff
@@ -1858,6 +1789,26 @@ pub enum DateTruncTo {
     Millennium,
 }
 
+impl fmt::Display for DateTruncTo {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.write_str(match self {
+            DateTruncTo::Micros => "microseconds",
```

Comment:
I know that we can't do anything about it but I strongly dislike that postgres makes it so that microseconds and milliseconds are plural and all the other units are singular :(

Reply:
I know, right? I'm 100% with you.